### DISCUSSION TITLE
Feature Request - Comprehensive Support for ACF Fields in Elementor

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#24774] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The current limitation in Elementor's support for Advanced Custom Fields (ACF) restricts the seamless integration of diverse ACF fields, hindering developers like me from fully leveraging ACF's capabilities. This feature request aims to enhance Elementor's compatibility with ACF, providing a more versatile solution for dynamic website designs.

![image](https://github.com/elementor/elementor/assets/99708427/6b2b0907-78aa-4814-8b7f-2d8a336e03a4)

![image](https://github.com/elementor/elementor/assets/99708427/9eca0bb3-e7b7-46ea-a8b0-f09338e78fde)

### Describe the solution you'd like.

Develop a native Elementor feature supporting various ACF fields, including repeater, user, taxonomy, etc. This would enable effortless integration and manipulation of ACF data within Elementor, expanding possibilities for website development.

### Describe alternatives you've considered.

Current alternatives involve custom coding or third-party plugins, but these lack the seamless integration and reliability of a native solution.

### Additional context

Native ACF integration in Elementor would streamline workflows, reduce reliance on external plugins, and enhance the overall developer experience, meeting the demand for flexibility in modern website development.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Fzyyyy** [comment]
+1
